### ISSUE TITLE
Forslag til rolle: Logningsansvarlig

### ISSUE THREAD

**Issue by mariushart** [JosefAssad/SeMaWi#182]
I fbm systemer kan det være praktisk at have anført, hvilken bruger der er ansvarlig for logs på systemet, særligt gennemgang af disse jf. forpligtelser ifht revision af adgang og brug af personlige oplysninger, som er lagret i organisationens systemer. Brugeren er ikke nødvendigvis sammenfaldende med Systemejer, men rollen bør defaulte til Systemejer, der så må udpege en ansvarlig.

Det bør diskuteres om den logningsansvarlige skal defineres pr. datasæt eller på systemniveau - men det er nok vigtigt, at den ansvarlige er tæt på de brugere, hvis anvendelse af data der revideres, da forretningsforståelse er central for at vurdere om der er tale om uhensigtsmæssig brug.

**Comment by JosefAssad** [comment]
@ClockEnd @baffioso tanker?

**Comment by baffioso** [comment]
Tænker at det for vores vedkommende giver mest mening at have det på
systemniveau.

Den 30/11/2016 17.38 skrev "Josef Assad" <notifications@github.com>:

**Comment by JosefAssad** [comment]
Yeah det er også der vi har implementeret hos Marius. Uanset om det er mere "korrekt" at logningsansvar kobles til system eller data, så forestiller jeg mig at størstedelen af verdenen ser logning som systemrelateret så det er nok det fornuftige sted at placere property'et.

Når datamodellem hos Marius er stabil så skal vi kigge sammen på den for at de om ikke vi kan forene metadatafelterne; her er logningsansvarlig et godt eksempel. Jeg står for followup. :)

**Comment by baffioso** [comment]
Carpe diem
